### PR TITLE
Paste as plain text in compose box

### PR DESCRIPTION
**Pull request by gmethvin**
@2is10 

Tested in chrome and firefox.

### CODE REVIEW

**Comment by eishay** [comment]
wow, cool!

File: packrat/scripts/compose.js [hunk 1]
```diff
@@ -58,6 +58,13 @@ function attachComposeBindings($c, composeTypeName, enterToSend) {
     $f[0].classList[this.firstElementChild === this.lastElementChild && !this.textContent ? "add" : "remove"]("kifi-empty");
   }).on("transitionend", function() {
     updateMaxHeight();
+  }).on("paste", function(e) {
+    var cd = e.originalEvent.clipboardData;
+    if (cd) {
+      e.preventDefault();
+      var html = $('<div>').text(cd.getData("text/plain")).html();
+      document.execCommand("insertHTML", false, html);
```
Comment (2is10) [inline packrat/scripts/compose.js]: Did you try the `"insertText"` command? It would allow us to skip the `<div>`.

**Comment by 2is10** [comment]
Nice. I didn’t know about `document.execCommand`!

File: packrat/scripts/compose.js (hunk 1 above)
Comment (gmethvin) [inline packrat/scripts/compose.js]: I'll try it

Comment (gmethvin) [inline packrat/scripts/compose.js]: Nice. It works.

**Pull request merged**